README.md

```diff
@@ -66,7 +66,60 @@ By definition, members work on their respective squad, although they are free to
+4. Total effort
+
+
+| Version                  | Release Engineer(s)              | QA effort              | Engineering effort | Total effort |
+|--------------------------|----------------------------------| ---------------------- |--------------------|--------------|
+| 3.2.0                    | Danilo Aliberti                  | Automated: `12h53`<br>Manual: `10h`<br>| `UA-8166: 4h`<br>`UA-8149: 2d`<br>`UA-8187: 3h`<br>| Total: **3d6h** |
+
+### Release step-by-step
+
+1. Cut a release branch when the sprint ends (typically when the new sprint starts on Monday). Create a new branch from develop and push to origin (e.g `release/3.2.0`).
+2. Bump the release version by triggering its command (eg. `/release babylon:3.2.0`) in `#ios-builds` (you can run the command every time you want to upload a new build).
+    1. This creates a test Tesflight build (try to do one as early as possible, so that you can catch issues like missing/expired certificates or profiles and any other production build errors early)
+3. Trigger a hockey build from that branch.
+4. Testers will then begin their work against the build you just created.
+5. Any hotfix should target that branch, you, as the release engineer, are responsible for double checking the hotfix PR is pointing to the release branch (instead of develop). 
+7. Create a board for the release. Use a filter to reduce its scope, eg `project = UA AND affectedVersion = "iOS 3.2.0"`.
+8. Create a new version in AppStoreConnect (login using your own account) / My Apps - on sidebar: + Version or Platform: iOS, input the version number.
+    1. Add the release notes and update the release notes document.
+    2. Add your release to the calendar.
+    3. Check if you need anything from the Marketing Team. (`#MarketingQuestions`)
+    4. Send the release notes to `#ClinicsOps`, so they have visibility on the release.
+9. Perform a quick exploratory test on the TestFlight build to make sure everything looks okay.
+(e.g. verifying that DigitalTwin Assets are visible and are not dropped due to Git LFS issues)
+10. By now, QA should be notified that there is a new version in TestFlight.
+11. When QA has signed off a particular build, submit the app to Apple.
+    1. Make sure the release is Manual.
+    2. When submitting to release, you are asked if the app uses the Advertising Identifier (IDFA). The answer is YES. You are then presented with three options please select as followed:
+    1. 🚫 Serve advertisements within the app
+    2. ✅ Attribute this app installation to a previously served advertisement
+    3. ✅ Attribute an action taken within this app to a previously served advertisement
+12 Once the app is accepted by Apple
+    1. Tag the release and upload the binary. (If you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
+    2. Merge the changes back to develop.
+13. Please fill the release calendar.
```

Comment:
It looks like either this one or https://github.com/Babylonpartners/ios-playbook/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R105 are redundant